skimage/filters/_fft_based.py

```diff
@@ -0,0 +1,108 @@
+import numpy as np
```

Comment:
```suggestion
import functools

import numpy as np
```

added functools for use below

Reply:
@din14970 the import order/grouping that @grlee77 suggested follows a convention: https://www.python.org/dev/peps/pep-0008/#imports

Reply:
woops, I made the changes locally and didn't see that when I was making corrections. I also added an empty line below the numpy import as per the PEP8 rule. I found some other files in the repo that do not strictly adhere to these rules though ;)